### PR TITLE
Fix comments with indents

### PR DESCRIPTION
**Pull request by greshilov**
Fix for #548.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 covdefaults
 coverage
 pytest
+ruamel.yaml
```
Comment (asottile) [inline requirements-dev.txt:4]: this is not correct

Comment (greshilov) [reply requirements-dev.txt]: Deleted.

I was unable to run tests without this package.

Comment (asottile) [reply requirements-dev.txt]: you probably have extremely old setuptools / pip / tox / virtualenv-- upgrade those

Comment (greshilov) [reply requirements-dev.txt]: Sure, looks like I misconfigured something.

**Review by asottile — APPROVED** [review]

**Review by MasterSlave420 — COMMENTED** [review]
Thanks

**Comment by ocalla22** [comment]
Thanks for adding this. 

@asottile do you know when the next release would be approximately?
There's a few additions I'd like to use, this included 😄

**Comment by asottile** [comment]
> 
> 
> Thanks for adding this.
> 
> @asottile do you know when the next release would be approximately?
> There's a few additions I'd like to use, this included 😄

the next release will be when the next release is

please don't ask for releases (from anyone, not just me), it's borderline rude -- if you want free work consider [sponsoring](https://github.com/sponsors/asottile)

**Comment by ocalla22** [comment]
@asottile , Sorry, about that.
 I’ve better informed myself and I now see that was very bad etiquette on my behalf. Thank you for correcting it :)
   I appreciate you and your work.
   
   Please accept my apology. :)

**Pull request merged**